app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -20,7 +20,7 @@ def authenticate
       @current_user.present?
     rescue Errors::AccessTokenExpiredError => e
       render json: { errors: e }, status: :forbidden
-    rescue Errors::StandardError => e
+    rescue Errors::StandardError, MPI::Errors::AccountLockedError => e
```

Comment:
You don't need to add AccountLockedError here, since StandardError will pick it up

Reply:
It wasn't picking it up in my testing, that's why I added it. I think because of the `SignIn` vs `MPI` as the implicit parent module of `Errors`? I'm still trying to figure this out.

Reply:
oh you know what? in my testing I was seeing 500s returned, so the error still logged the user out, it just wasn't getting rescued.

I think the functions in `authentication.rb` are wrong, they should be rescuing the base `Errors::StandardError`, not the `SignIn::Errors::StandardError`